sklearn/feature_selection/_sequential.py

```diff
@@ -144,6 +146,31 @@ class SequentialFeatureSelector(SelectorMixin, MetaEstimatorMixin, BaseEstimator
     (150, 3)
     """
 
+    _parameter_constraints = {
+        "estimator": [HasMethods(["fit"])],
+        "n_features_to_select": [
+            StrOptions({"auto", "warn"}, deprecated={"warn"}),
+            Interval(Real, 0, 1, closed="right"),
+            Interval(Integral, 0, None, closed="neither"),
+            Hidden(None),
+        ],
+        "tol": [None, Interval(Real, 0, None, closed="neither")],
+        "direction": [StrOptions({"forward", "backward"})],
+        "scoring": [
+            None,
+            StrOptions(set(get_scorer_names())),
+            callable,
+            list,
+            dict,
+            tuple,
+        ],
+        "cv": [
+            "cv_object",
+            callable,
+        ],
```

Comment:
I do not think `cv` accepts a callable and `cv_object` is enough to handle all the accepted parameters for `cv`:


```suggestion
        "cv": ["cv_object"],
```